src/subscriptions/SubscriptionScrollView.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Card, Badge, Scrollable } from '@edx/paragon';
+
+const SubscriptionScrollView = ({
+  subscriptions = [
+    {
+      title: 'Blockchain Fundamentals',
+      org: 'University of California, Berkeley',
```

Comment:
rename to `organizations`: []; and this will be `array` but you can confirm with Shafqat